cmd/wego/flux/cmd.go

```diff
@@ -0,0 +1,63 @@
+package flux
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/BurntSushi/toml"
+	"github.com/spf13/cobra"
+)
+
+//go:embed bin/flux
+var fluxExe []byte
+
+type dependencies struct {
+	Flux flux
+}
+type flux struct {
+	Version string
+}
+
+var Cmd = &cobra.Command{
+	Use:   "flux",
+	Short: "Use flux commands",
+	Run:   runCmd,
+}
+var path = "temp/bin"
+var exePath string
+
+func init() {
+	var dep dependencies
+	if _, err := toml.DecodeFile("tools/dependencies.toml", &dep); err != nil {
+		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
```

Comment:
This is really a nit, but we have exactly the same error handling in a bunch of places. Since we're exiting when we get an error, we could wrap that in a function. Something like:

```
func checkError(err error) {
   if err != nil {
      fmt.Fprintf(os.Stderr, "Error: %v\n", err)
      os.Exit(1)
   }
}

and then:

checkError(toml.DecodeFile("tools/dependencies.toml", &dep))
exePath = path + "/flux-" + dep.Flux.Version

if _, err := os.Stat(path); os.IsNotExist(err) {
   checkError(os.MkdirAll(path, 0755))
   checkError(os.WriteFile(exePath, fluxExe, 0755))
}
```